core/src/main/scala/cromwell/core/simpleton/WdlValueSimpleton.scala

```diff
@@ -26,6 +27,7 @@ object WdlValueSimpleton {
   implicit class WdlValueSimplifier(wdlValue: WdlValue) {
     def simplify(name: String): Iterable[WdlValueSimpleton] = wdlValue match {
       case prim: WdlPrimitive => List(WdlValueSimpleton(name, prim))
+      case expr: WdlExpression => List(WdlValueSimpleton(name, WdlString(expr.valueString)))
```

Comment:
Some symbols are stored as `WdlExpression` so I got a match error when trying to simpletonize them. I can filter them out earlier before calling `simplify` if that's a problem, but `simplify` should at least handle this case even if it's just throwing an exception